README.md

```diff
@@ -161,6 +161,15 @@ the `encryption_*` and `ssl_*` options
 directly to the constructor as keyword arguments, even though it is
 required when they are placed in the environment file.
 
+pam (pam_password) logins
+-------------------------
+
+Starting with v2.0.0, the python iRODS client is able to authenticate under PAM using the same file-based client environment as does
+any of the iCommands.
+
+Caveat for iRODS 4.3+: when upgrading from 4.2, the "irods_authentication_scheme" setting must be changed from "pam" to "pam_password" in
+`~/.irods/irods_environment.json` for all file based client environments (except that of the iRODS service account).
```

Comment:
A couple of things...

1. Maybe add a note indicating that this change is necessary for iCommands as well. In other words, the pam -> pam_password transition is not special for PRC.
2. The iRODS service account note may not be necessary. Most deployments use native authentication for the service account's iRODS user authentication, but if the service account does use pam/pam_password, then the service account's client environment would need to be updated, too.

Reply:
After some discussion, @d-w-moore and I decided that we should just remove the parenthetical statement and the rest is fine as-is.

Reply:
Resolved.